code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -57,66 +62,58 @@
 	set src in view(1)
 
 	if(usr.incapacitated() || !istype(usr, /mob/living))
```

Comment:
I'll replace it with can_use_topic, so TK people would be able to pull the cards out remotely.